javatests/arcs/core/host/test.arcs

```diff
@@ -5,9 +5,15 @@ particle TestProdParticle in 'arcs.core.host.TestProdParticle'
 
 particle TestHostParticle in 'arcs.core.host.TestHostParticle'
 
+particle TestReflectiveParticle in 'arcs.core.host.TestReflectiveParticle'
+  data: reads [Thing {name: Text}]
+
 recipe TestRecipe
```

Comment:
As commented in the earlier PR - you should have your own recipe for this test you are introducing, this way you don't need need to add registrations for TestProdParticle and TestHostParticle which do not do anything in your test.

Reply:
Can you address this please?

Reply:
Just pushed the change.